app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -310,6 +310,20 @@ def prepare_disk(disk_spec, ems_storage_uid)
     }
   end
 
+  def vm_migrate(vm, options = {})
+    host_id = URI(options[:host]).path.split('/').last
+
+    mig_options = {
```

Comment:
What is mig?  Is that migration?  If so, prefer spelling it out for readability/clarity.